microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md

```diff
@@ -208,14 +208,14 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 |Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
 |------------|--------------|
-|Restore-AzureADDeletedApplication|[Restore-MgApplication](/powershell/module/microsoft.graph.applications/restore-mgapplication?view=graph-powershell-1.0&preserve-view=true)|
+|Restore-AzureADDeletedApplication|[Restore-MgDirectoryDeletedItem](/powershell/module/microsoft.graph.identity.directorymanagement/restore-mgdirectorydeleteditem?view=graph-powershell-1.0&preserve-view=true)|
 
 ### Devices
 
 |Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
 |------------|--------------|
 |Add-AzureADDeviceRegisteredOwner|[New-MgDeviceRegisteredOwnerByRef](/powershell/module/microsoft.graph.identity.directorymanagement/new-mgdeviceregisteredownerbyref?view=graph-powershell-1.0&preserve-view=true)|
-|Add-AzureADDeviceRegisteredUser|[New-MgDeviceRegisteredUserByRef](/powershell/module/microsoft.graph.identity.directorymanagement/new-mgdeviceregistereduserbyref?view=graph-powershell-1.0&preserve-view=true)|
+|Add-AzureADDeviceRegisteredUser||
```

Comment:
@msewaweru - For consideration next time, rather than leaving it blank, would it be possible for us to put something else like a link to some roadmap that shows when some modules will have a Graph PowerShell equivalent or a link to an article explaining why some have Graph PS equivalents and some don't?

Reply:
Great idea, @CelesteDG. I will work closely with the team to get such updates for the cmdlets missing equivalents.